api/db/populate-events.js

```diff
@@ -0,0 +1,104 @@
+const txtgen = require('txtgen');
+const { 
+  uniqueNamesGenerator,
+  adjectives,
+  animals,
+  starWars,
+  colors
+} = require('unique-names-generator');
+const { 
+  createPrompt,
+  getRandomDate,
+  getCountry,
+  getRegion ,
+  getRandomInterests
+} = require('./utils');
+
+const Interest = require('../models/Interest');
+const User = require('../models/User');
+const Event = require('../models/Event');
+
+const createEvent = async (eventParams) => {
+  const event = new Event({
+    name: eventParams.name,
+    description: eventParams.description,
+    startDate: eventParams.startDate,
+    endDate: eventParams.endDate,
+    country: eventParams.country,
+    region: eventParams.region,
+    createdBy: eventParams.createdBy,
+    relatedInterests: eventParams.relatedInterests,
+    coverPhoto: eventParams.coverPhoto,
+    attendantsLimit: eventParams.attendantsLimit,
+  });
+  await event.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New event created: ${event}`);
+  }
+};
+
+const createEvents = async (multiplier, randomLocation, verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating events collection...');
+
+  let usersList = await User.distinct('_id');
+  if (usersList.length === 0) {
+    console.log(
+      '\x1b[33m',
+      'The users collection does not exist. Aborted events collection population.'
+    );
+    return;
+  }
+
+  let interestsList = await Interest.distinct('_id');
+  if (interestsList.length === 0) {
+    const proceed = await createPrompt(
+      'The interests collection does not exist. Events created will not have any related interests. Do you want to continue?'
+    );
+    if (!proceed) {
+      console.log('\x1b[33m', 'Aborted events collection population');
+      return;
+    }
+  }
+
+  displayLogs = verbose;
+  const eventsArray = [];
+  const now = new Date();
+  const futureDate = new Date().setFullYear(now.getFullYear() + 5);
+
+  for (let i = 0; i < 50 * multiplier; i++) {
+    const name = uniqueNamesGenerator({
+      dictionaries: [adjectives, animals, colors, starWars],
+      length: 3,
+      style: 'capital',
+      separator: ' '
+    });
+
+    const description = txtgen.article(Math.floor(Math.random() * 15));
```

Comment:
```suggestion
    const description = txtgen.article(Math.floor(Math.random() * 2) + 1);
```
Maybe 15 is too much, as that will look huge! And 0 would make the event invalid.